### PR TITLE
ENDOC-161 Add the Application Wizard page.

### PR DESCRIPTION
**Pull request by nshaw**

### CODE REVIEW

**Comment by nshaw** [comment]
Language may change slightly based on questions I have sent to Product but this should be close.

File: vuepress/docs/next/tutorials/cms/app-builder/hello-world.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ development environment you would build and deploy widgets differently,
 however this example provides a quick idea of the building blocks.
 
 1. In the App Builder menu, go to: `Components → Micro frontends & Widgets`
-    -   At bottom of the page, select ADD 
+    -   At bottom of the page, selectADD
```
Comment (joewhite101) [inline vuepress/docs/next/tutorials/cms/app-builder/hello-world.md:15]: Missing a space. Maybe unintentional edit.

**Review by joewhite101 — CHANGES_REQUESTED** [review]
One small note.

File: vuepress/docs/next/tutorials/cms/app-builder/hello-world.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/next/tutorials/cms/app-builder/hello-world.md]: Good catch, fixed.

**Review by joewhite101 — APPROVED** [review]
LGTM

**Pull request merged**